plemiona_api/utils.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework.exceptions import APIException
+from rest_framework.views import exception_handler
+
+def custom_exception_handler(exc, context):
+    response = exception_handler(exc, context)
+
+    try:
+        if response is not None and isinstance(exc, APIException):
+            data = {"errors": {}}
+            for field, errors in exc.detail.items():
+                for error in errors:
+                    data["errors"][field.capitalize()] = [error.capitalize()]
+            response.data = data
```

Comment:
In the previous version of errors handling we were using such a pattern: [Capitalized error type] + [rest of the error statement]. I understand that finally we are going to use this format. Am I right?

Reply:
It's the current format
```json
{
    "errors": {
        "Player": [
            "Nickname already in use."
        ]
    }
}
```
And as long as I tested, everything seems to look the same